ts/kpt-functions/src/testing.ts

```diff
@@ -83,7 +83,7 @@ export class TestRunner {
     expectedErrorMessage?: string | RegExp
   ): () => Promise<void> {
     return async () =>
-      await this.assert(
```

Comment:
why remove 'await'? This actually will not wait for function invocation to complete anymore.

On a different note: We should not mix this subtle changes with other unrelated changes like docs.

Reply:
Removing await, let -> const, etc. are all auto-fixed tslint errors. The docs changes are related. For example, _types_.html links to line numbers in source code which would otherwise be out of date.

Reply:
To give more detail, this is the no-return-await rule (https://eslint.org/docs/rules/no-return-await). Since we already say return async, we don't need await.